client/assets/sass/styles.sass

```diff
@@ -4,7 +4,6 @@
 @import '_bem-support/index'
 
 // Variables: colors, sizes, ...
-@import 'colors'
 @import 'app_colors'
 @import 'app_urls'
 
```

Comment:
weary eyes might be missing something, but why not include `../../app/shared` in here and remove the` import 'app_colors'` from each file in shared that requires colors?